flare_portal/experiments/templates/experiments/participant_list.html

```diff
@@ -131,7 +161,17 @@ <h3 class="card-title">Participants</h3>
 
                         <div class="card-footer d-flex">
                             <button class="btn btn-sm btn-secondary" @click.prevent="forms.push('')">Add participants</button>
-                            <button class="btn btn-sm btn-primary ml-auto" type="submit">Save changes</button>
+                            <button x-show="selectedRows.length == 0" class="btn btn-sm btn-primary ml-auto" type="submit">Save changes</button>
+                            <button
+                                x-show="selectedRows.length > 0"
+                                x-text="selectedRows.length > 1 ? `Delete ${selectedRows.length} participants` : `Delete participant`"
+                                class="btn btn-sm btn-danger ml-auto"
+                                type="button"
+                                @click="
+                                    window.location = `{% url 'experiments:participant_delete_batch' project_pk=view.kwargs.project_pk experiment_pk=view.kwargs.experiment_pk participant_pks='PKS-PLACEHOLDER' %}`
+                                        .replace('PKS-PLACEHOLDER', selectedRows.join())
+                                "
+                            ></button>
```

Comment:
Same as the other view – this really should be an `a` tag, and it likely needs URL encoding for `selectedRows.join()`.